source/enumerate-indexes.rst

```diff
@@ -379,6 +386,9 @@ The shell implements the first algorithm for falling back if the
 
 Version History
 ===============
+0.5.0 - 2020-01-14
+    Update to mention handling of the ``ns`` field being removed from the
+    response to the ``listIndexes`` server command.
```

Comment:
I think it'd helpful to note the server version here. We can probably also make this more concise. Consider:

> MongoDB 4.4 no longer includes ``ns`` field in ``listIndexes`` responses.

Reply:
done